test/fixtures/users.yml

```diff
@@ -0,0 +1,45 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+user1:
+  username: username1
+  date_joined: 2017-10-11
+
```

Comment:
Pro tip: you can use erb inside a fixtures file. This could be shortened to something like

```ruby
<% 20.times do |i| %>
user_<%= i %>:
  username: username_<%= i %>
  date_joined: 2017-10-<%= i %>
<% end %>
```